blocks/social/social.js

```diff
@@ -1,6 +1,23 @@
 import { getMetadata } from '../../scripts/lib-franklin.js';
 import { decorateAnchors, getLanguage, fetchTagsOrCategories } from '../../scripts/scripts.js';
 
+const getSocialPaths = (type) => {
```

Comment:
Nit maybe a better name would be `getSocialURL()` as it returns a full URL and also it returns just 1 thing.